comps/comps-foreman-rhel7.xml

```diff
@@ -370,7 +370,7 @@
       <packagereq type="default">puppet-agent-puppet-strings</packagereq>
       <packagereq type="default">puppet-agent-rgen</packagereq>
       <packagereq type="default">puppet-agent-yard</packagereq>
-      <packagereq type="default">python2-keycloak-httpd-client-install</packagereq>
+      <packagereq type="default">python3-keycloak-httpd-client-install</packagereq>
```

Comment:
Shouldn't EL7 remain on Python 2? At least the spec suggests so

Reply:
Changed.